packages/verified-fetch/src/index.ts

```diff
@@ -90,7 +90,7 @@
  *
  * The [helia](https://www.npmjs.com/package/helia) module is configured with a libp2p node that is suited for decentralized applications, alternatively [@helia/http](https://www.npmjs.com/package/@helia/http) is available which uses HTTP gateways for all network operations.
  *
- * You can see variations of Helia and js-libp2p configuration options at <https://helia.io/interfaces/helia.index.HeliaInit.html>.
+ * You can see variations of Helia and js-libp2p configuration options at <https://helia.io/interfaces/helia.HeliaInit.html>.
```

Comment:
```suggestion
 * You can see variations of Helia and js-libp2p configuration options at <https://ipfs.github.io/helia/interfaces/helia.HeliaInit.html>.
```

api docs have moved to `ipfs.github.io/helia`